src/components/NavbarStudy.scss

```diff
@@ -0,0 +1,170 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
```

Comment:
index.css has this style.

Reply:
Yes, but why should I depend on the second style file?

Reply:
And index.css hasn't for  ' * ' margin: 0; and padding: 0; this file has only box-sizing: border-box;

Reply:
done